res/skins/Deere/style-mac.qss

```diff
@@ -7,7 +7,7 @@
 }
 
 
-/* hack to hide the checkmark, otherwise text gets cut off
+/* Previously necessayr hack to hide the checkmark, otherwise text gets cut off
```

Comment:
Why not just delete the code?

Reply:
For documentation. It took a while to notice and figure out the fix, so its valuble info IMO

Reply:
I'd rather have the code deleted if it is no longer necessary. It is still in the Git history if that info is needed later.